### PR TITLE
Riscv zce extension tablejump

### PR DESCRIPTION
**Pull request by ScottEgerton**
This is the current state of the table jump work. This is an implementation of the older specification and has not yet been updated to reflect the latest specification changes. I’m mostly posting this so that others can access and review this. There are a few known minor test failures, however these are only in newly added tests and are not regressions. Currently this does not contain a cost function.

### CODE REVIEW

File: lld/ELF/Arch/RISCV.cpp [hunk 1]
```diff
@@ -260,9 +295,40 @@ RelExpr RISCV::getRelExpr(const RelType type, const Symbol &s,
   }
 }
 
-// Extract bits V[Begin:End], where range is inclusive, and Begin must be < 63.
-static uint32_t extractBits(uint64_t v, uint32_t begin, uint32_t end) {
-  return (v & ((1ULL << (begin + 1)) - 1)) >> end;
+void RISCV::processSection(InputSection &inputSection) const {
```
Comment (Xinlong-Wu) [inline lld/ELF/Arch/RISCV.cpp:298]: What does this empty function for?

**Review by Xinlong-Wu — COMMENTED** [review]
This implementation is for V0.50.
we are waiting for V0.70 update

**Comment by abukharmeh** [comment]
@ScottEgerton  Any updates on this one ?

**Comment by Xinlong-Wu** [comment]
the patch was rebased & merged from the internal repo, close this pr.

**Pull request closed without merging**